traffic_ops/testing/api/v2/tc-fixtures.json

```diff
@@ -2572,32 +2572,6 @@
             "type": "STEERING_WEIGHT"
         }
     ],
-    "jobs": [
-        {
-            "dsName": "ds1",
-            "request": {
-                "startTime": "2019-01-01 00:00:00",
-                "ttl": 80,
-                "regex": "/foo"
-            }
-        },
-        {
-            "dsName": "ds2",
```

Comment:
removing this test data is going to break the tests that depend on them -- they might not fail, since they don't always check to make sure their test data exists, but the tests won't actually test what they were meant to test

Reply:
You may be correct, regardless I meant to put that it a separate PR.